src/Plugin/Derivative/LocalTasks.php

```diff
@@ -37,7 +37,7 @@ public function getDerivativeDefinitions($base_plugin_definition) {
     foreach ($uf_groups as $key => $uf_group) {
       if ($uf_group['is_active']) {
         $this->derivatives["civicrm.{$key}"] = $base_plugin_definition;
-        $this->derivatives["civicrm.{$key}"]['title'] = $uf_group['title'];
+        $this->derivatives["civicrm.{$key}"]['title'] = $uf_group['frontend_title'];
```

Comment:
@adixon this broadly is heading down the right path but I would suggest doing this instead

```suggestion
        !empty($uf_group['frontend_title']) ? $uf_group['frontend_title'] : $uf_group['title'];
```

Reply:
Yes, that's a good idea. I switched the logic around to save that exclamation mark and my logic processing brain.